### DISCUSSION TITLE
Pyarmor-7 pack command strip away important traceback details

### DISCUSSION THREAD

**Discussion by FuckingToasters** [dashingsoft/pyarmor#1789] [Q&A] [answered]
Hello, so i used following pyarmor command to obfuscate the script:

```bat
cd D:\workspace\joinify
C:\Users\Infinimonster\venv\Scripts\activate && pyarmor-7 pack -s Joinify.spec --clean -e " --add-binary 'c:\users\infinimonster\appdata\local\programs\python\python39\lib\site-packages\tls_client\dependencies\tls-client-64.dll;tls_client\dependencies' --onefile --icon=icon.ico" -x " --enable-suffix --advanced 4 --restrict 102 --obf-mod 0 --obf-code 2" Joinify.py
```

I also tried removing every parameter but sadly the issue is the same.

I would like to show the same error amount which shows when running the Joinify.py file directly. 

I use the rich libary to save beautified Traceback info as HTML File but the output only contain the function name when running the comopiled executable. Not even the file etc.

**Comment by jondy** [comment]
The traceback couldn't be same as original. But please show an example, for example, what's the expected traceback and what's the real traceback?

**Reply by FuckingToasters** [reply]
Below i attached 1 screenshots of the error. The Screenshot with more details happen when running the .py file. The one with less details happens when running the compiled .exe


![CrosshairX_Jw9cwXSZIN](https://github.com/dashingsoft/pyarmor/assets/94435104/e8c4e65b-f2e5-4edd-ac0f-53568c58c452)


![CrosshairX_vMLLwt15GD](https://github.com/dashingsoft/pyarmor/assets/94435104/6ff5339f-2f49-474d-837a-500069007cec)

**Comment by jondy** [comment, marked as answer]
How about no obfuscated scripts? Just pack the original script by PyInstaller, does .exe raise the same traceback as running the .py file?
If not, it's PyInstaller issue.

**Reply by FuckingToasters** [reply]
> How about no obfuscated scripts? Just pack the original script by PyInstaller, does .exe raise the same traceback as running the .py file? If not, it's PyInstaller issue.

Helo, i now used pyinstaller only. The error still shows less detail than running the  normal .py file. I now opend a Issue on Pyinstaller Reponsetory and close this one as completed. Should've done that before lmao

# https://github.com/orgs/pyinstaller/discussions/8567
